CHANGES.rst

```diff
@@ -11,6 +11,65 @@ Changes
 -------
 
 - Drop support for Python 3 versions older than Python 3.6. (#341)
+- Re-export CorePlugin in envisage.api (#332)
+- reate and fill plugin subpackage api modules (#323)
+- Add relevant classes to envisage.ui.tasks.api (#322)
+- Make TasksApplication.gui expect an IGUI interface, not a GUI instance (#301)
+- Deprecate safeweakref and replace its uses (#275)
+
+Fixes
+-----
+
+- Fix unregister_service ValueError bug (#345)
+- Fix the MOTD example (#319)
+- Fix the Hello_World example (#318)
+- Fix the attractors tasks application example (#317)
+
+Features
+--------
+
+- Add a new shell click command (#305)
+
+Documentation
+-------------
+
+- Setup intersphinx in docs (#343)
+- Add documentation for envisage APIs (#340)
+- Use jinja templates for API documentation (#339)
+- Improve API docs : document traits (#334)
+- Rebuild documentation, mostly to fix search functionality (#290)
+
+Removals
+--------
+
+- Remove single_project (#331)
+- Remove class_load_hooks and ClassLoadHook (#321)
+
+Tests
+-----
+
+- Add tests for ExtensionRegistry getters (#349)
+- Add tests to demonstrate behaviour when mutating extension point directly
+  (#346)
+- Use mixin instead of having ProviderExtensionRegistryTestCase inherit from
+  ExtensionRegistryTestCase (#335)
+- TST: Switch on default warning flag for CI test command (#326)
+- Add test eggs for Python 3.9; remove eggs for Python 2.7 (#289)
+
+Build
+-----
+
+- Add apptools to source_dependencies in etstool.py (#348)
```

Comment:
While I read this it did not occur to me that 'source-dependencies' is only meant for cron job, so I thought envisage had omitted to declare apptools as a dependency (e.g. the one in setup.py), but that's not what this is about. Maybe something like this?
```suggestion
- Fix CI cron job setup to install apptools (#348)
```